src/repos/UserSessionRepo.ts

```diff
@@ -45,13 +42,7 @@ const createOrUpdateSession = async (
  * @return {SerializedUserSession} Contains session information for user
  */
 const createUserAndInitializeSession = async (
-  clubs: Club[],
-  graduationYear: string,
-  hometown: string,
-  interests: Interest[],
-  login: LoginTicket,
-  major: CornellMajor,
-  pronouns: string
+  login: LoginTicket
```

Comment:
is this actually called a loginticket? interesting.. i would have thought that it's logintoken but i learn new things every day

Reply:
Yes indeed, it is a type defined in the `google auth` library. I agree, it's wild stuff.